src/GuppyDataExplorer/GuppyDataExplorer.jsx

```diff
@@ -35,12 +37,21 @@ class GuppyDataExplorer extends React.Component {
           onReceiveNewAggsData={this.handleReceiveNewAggsData}
           onFilterChange={this.handleFilterChange}
           rawDataFields={this.props.tableConfig.fields}
+          accessibleFieldCheckList={this.props.guppyConfig.accessibleFieldCheckList}
         >
-          <ConnectedFilter
+          <ExplorerTopMessageBanner
+            className='guppy-data-explorer__top-banner'
+            tierAccessLevel={this.props.tierAccessLevel}
+            tierAccessLimit={this.props.tierAccessLimit}
+            guppyConfig={this.props.guppyConfig}
+            getAccessButtonLink={config.dataExplorerConfig.getAccessButtonLink}
+          />
+          <ExplorerFilter
             className='guppy-data-explorer__filter'
-            filterConfig={this.props.filterConfig}
-            guppyConfig={{ type: this.props.guppyConfig.dataType, ...this.props.guppyConfig }}
-            fieldMapping={this.props.guppyConfig.fieldMapping}
+            guppyConfig={this.props.guppyConfig}
+            getAccessButtonLink={config.dataExplorerConfig.getAccessButtonLink}
```

Comment:
Move this `getAccessButtonLink` to `/GuppyDataExplorer/index.jsx` to keep all config items in one place?